exercises/house/canonical-data.json

```diff
@@ -0,0 +1,322 @@
+            "This is the farmer sowing his corn",
+            "that kept the rooster that crowed in the morn",
+            "that woke the priest all shaven and shorn",
+            "that married the man all tattered and torn",
+            "that kissed the maiden all forlorn",
+            "that milked the cow with the crumpled horn",
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "verse 12",
+          "property": "verse",
+          "verse": 12,
+          "expected": [
+            "This is the horse and the hound and the horn",
+            "that belonged to the farmer sowing his corn",
+            "that kept the rooster that crowed in the morn",
+            "that woke the priest all shaven and shorn",
+            "that married the man all tattered and torn",
+            "that kissed the maiden all forlorn",
+            "that milked the cow with the crumpled horn",
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "multiple verses",
+          "property": "verses",
+          "start verse": 4,
+          "end verse": 8,
+          "expected": [
+            "This is the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built.",
+            "",
+            "This is the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built.",
+            "",
+            "This is the cow with the crumpled horn",
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built.",
+            "",
+            "This is the maiden all forlorn",
+            "that milked the cow with the crumpled horn",
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built.",
+            "",
+            "This is the man all tattered and torn",
+            "that kissed the maiden all forlorn",
+            "that milked the cow with the crumpled horn",
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "full rhyme",
+          "property": "verses",
+          "start verse": 1,
+          "end verse": 12,
```

Comment:
This requires knowing that there are 12 verses.
Is there a better way we can ask for all verses without having to know how many there are?